### DISCUSSION TITLE
Reading .sld file

### DISCUSSION THREAD

**Discussion by nasrul29** [Mapsui/Mapsui#1642] [Q&A]
Hi, i have been doing integration work where i need to migrate shapefiles from few sources and notice they came with .sld files and i just can't think of a way to use it..is geoserver any help because i have never use it before

**Comment by pauldendulk** [comment]
What kind of functionality are you trying to achieve exactly?

**Reply by nasrul29** [reply]
I'm trying to use the styles defined in the .sld files, for example if there is image for point, it will automatically loaded into my map
